File: src/main/java/fr/lewon/bot/manager/mappers/BotInfosMapper.java
package fr.lewon.bot.manager.mappers;

import java.util.List;

import org.mapstruct.Mapper;

import fr.lewon.bot.manager.entities.BotInfosDTO;
import fr.lewon.bot.manager.modele.RunnerInfos;

@Mapper(componentModel = "spring")
public interface BotInfosMapper {
	
	public List<BotInfosDTO> runnerInfosToDto(List<RunnerInfos> runnerInfos);

	public BotInfosDTO runnerInfosToDto(RunnerInfos runnerInfos);
	
}

File: src/main/java/fr/lewon/bot/manager/mappers/BotInfosMapper.kt
package fr.lewon.bot.manager.mappers

import fr.lewon.bot.manager.entities.BotInfoDTO
import org.mapstruct.Mapper

@Mapper(componentModel = "spring")
interface BotInfosMapper {
    fun runnerInfosToDto(runnerInfos: MutableList<RunnerInfos?>?): MutableList<BotInfoDTO?>?
    fun runnerInfosToDto(runnerInfos: RunnerInfos?): BotInfoDTO?
}